operations-guide/setting-up-axon-server/development-mode.md

```diff
@@ -0,0 +1,17 @@
+# Development Mode
+
+You can start Axon Server in development mode which enables some features for development convenience.
+
+These features can be enabled by configuring the following property:
+
+```text
+axoniq.axonserver.devtools.enabled=true
```

Comment:
suggestion: `axoniq.axonserver.devmode.enabled=true` (as the description calls it development mode)